independent-publisher-connectors/Fitbit/apiProperties.json

```diff
@@ -0,0 +1,29 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "fitbit",
```

Comment:
IdentityProvider should be anyone mentioned in the below schema definition. Please update. 

https://github.com/microsoft/PowerPlatformConnectors/blob/dev/schemas/paconn-apiProperties.schema.json#L299